### ISSUE TITLE
flip repo to private 

### ISSUE THREAD

**Issue by davidmooreppf** [yescomment/bh-flaminghydra-ghosttheme#98]
The goal is to do some site enhancements, which is work that we don't need to be public. 

So we might as well make this repo private, yeah. I don't see any reason why not. 

Ensuring Shirley and Maria's github handles are in this organization too.

**Comment by unitof** [comment]
My original thinking was that we could contribute back to Ghost and/or other journalists moving to the platform. But yeah, agrees Issues (and much of the code) is now quite internal and specific.

**Comment by unitof** [comment]
Ugh. That's also why. Will put in a ticket to have them manually [detach our fork](https://support.github.com/request/fork).
<img width="796" alt="80780" src="https://github.com/user-attachments/assets/a08af9bf-87dd-4ac1-8f40-86c15bd55ad6">

**Comment by davidmooreppf** [comment]
well we're flexible. Should we do the site enhancements in another repo?

**Comment by unitof** [comment]
Nope! Whole convenience is having it all together. We can always separately publish/opensource bits down the road if we want.

Request submitted, it's a standard process.

And I will say the first time I've had a helpful automated chat experience/flow.
<img width="897" alt="44425" src="https://github.com/user-attachments/assets/f1634299-f0a3-47a4-a676-cbb7e45f6b51">

**Comment by unitof** [comment]
[Ticket 2889181](https://support.github.com/ticket/2889181)
